source/_integrations/hunterdouglas_powerview.markdown

```diff
@@ -32,6 +33,98 @@ There is currently support for the following device types within Home Assistant:
 
 {% include integrations/config_flow.md %}
 
+## Shades
+
+Known devices and their supported functionality is listed below.
+
+<div class="note">
+If your shade is not listed please raise a feature request on the community forum.
+</div>
+
+| Type and Name                            | Capabilities                    |
+| :--------------------------------------- | :------------------------------ |
+| 4   - Roman                              | Bottom Up                       |
+| 5   - Bottom Up                          | Bottom Up                       |
+| 6   - Duette                             | Bottom Up                       |
+|  23 - Silhouette                         | 90° Tilt when closed            |
+|  42 - M25T Roller Blind                  | Bottom Up                       |
+|  44 - Twist                              | 180° Tilt when closed           |
+|  49 - AC Roller                          | Bottom Up                       |
```

Comment:
```suggestion
| Roman (4)                             | Bottom Up                       |
| Bottom Up (5)                          | Bottom Up                       |
| Duette (6)                            | Bottom Up                       |
| Silhouette (23)                        | 90° Tilt when closed            |
| M25T Roller Blind (42)                 | Bottom Up                       |
| Twist (44)                              | 180° Tilt when closed           |
| AC Roller  (49)                         | Bottom Up                       |
```

I think most people won't know the product type number so maybe that should come after the name (same below)